web/src/components/SpeakerCard.vue

```diff
@@ -0,0 +1,113 @@
+<template>
```

Comment:
Wait, I just realize this is a whole new file. The change on `src/components/speaker/SpeakerCard.vue` looks good to me, but why do we have this duplicate `SpeakerCard.vue` here?

Reply:
I made a merge mistake... lemme check

Reply:
Okay fixed. Good catch!